cloud-providers/aws/beanstalk/payara-server-docker-aws-eb/Dockerfile-Payara-Micro

```diff
@@ -0,0 +1,7 @@
+FROM payara/micro
+
+COPY target/application.war $DEPLOY_DIR
+
+EXPOSE 8080
```

Comment:
Expose already done in _payara/micro_ Docker Image, can be removed.

Reply:
Same comment as for Payara Server - I'd rather leave it as exposing 8080 by Payara Micro image is not documented.